src/app/app.module.ts

```diff
@@ -10,7 +9,7 @@ import { NavBarComponent } from './nav-bar/nav-bar.component';
 import { UserAuthModule } from './user-auth/user-auth.module';
 import { HomePageComponent } from './home-page/home-page.component';
 import { MatIconModule } from '@angular/material/icon'
-import { Routes } from '@angular/router';
+import { routing } from './app-routing.module';
```

Comment:
Why did you erase the AppRoutingModule to add routing?

Reply:
routing is being exported from the app-routing.module.ts and it includes the RouterModule which handles the routing. There is no need to use AppRoutingModule.